Scenes/Screens/RevivalShop/RevivalShop.tscn

```diff
@@ -3,450 +3,462 @@
-margin_left = -8.35962
-margin_top = -3.25937
-margin_right = 11.6404
-margin_bottom = 10.7406
-rect_scale = Vector2( 2, 2 )
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="Button" type="Button" parent="Control/MarginContainer/VBoxContainer/CenterContainer/VBoxContainer/Item_basic_hp"]
+margin_left = 155.0
+margin_right = 175.0
+margin_bottom = 20.0
 text = "+"
-align = 1
-valign = 1
-
-[node name="Item_items_price" type="Node2D" parent="ColorRect"]
-editor/display_folded = true
-position = Vector2( -243.973, -194.426 )
-
-[node name="Panel" type="Panel" parent="ColorRect/Item_items_price"]
-modulate = Color( 0.690196, 0.690196, 0.690196, 1 )
-margin_left = 332.0
-margin_top = 424.757
-margin_right = 725.857
-margin_bottom = 449.757
-
-[node name="Info" type="Label" parent="ColorRect/Item_items_price/Panel"]
-margin_left = -75.976
-margin_top = 2.0
-margin_right = 305.024
-margin_bottom = 23.0
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="Item_items_price" type="HBoxContainer" parent="Control/MarginContainer/VBoxContainer/CenterContainer/VBoxContainer"]
+margin_top = 192.0
+margin_right = 352.0
+margin_bottom = 212.0
+
+[node name="Info" type="Label" parent="Control/MarginContainer/VBoxContainer/CenterContainer/VBoxContainer/Item_items_price"]
+margin_top = 3.0
+margin_right = 289.0
+margin_bottom = 17.0
 text = "zmniejszenie kosztu przedmiotów u kupca o:"
 align = 2
 valign = 1
-
-[node name="Count" type="Label" parent="ColorRect/Item_items_price/Panel"]
-margin_left = 307.0
-margin_top = 2.0
-margin_right = 343.0
-margin_bottom = 17.0
-rect_scale = Vector2( 1.5, 1.5 )
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="Count" type="Label" parent="Control/MarginContainer/VBoxContainer/CenterContainer/VBoxContainer/Item_items_price"]
+margin_left = 293.0
+margin_right = 328.0
+margin_bottom = 20.0
+rect_min_size = Vector2( 35, 0 )
+size_flags_vertical = 1
 custom_colors/font_color = Color( 1, 0.937255, 0, 1 )
 text = "10%"
 align = 1
 valign = 1
+__meta__ = {
+"_edit_use_anchors_": false
+}
 
-[node name="Button" type="Button" parent="ColorRect/Item_items_price"]
-margin_left = 696.16
-margin_top = 426.746
-margin_right = 718.16
-margin_bottom = 446.746
-
-[node name="Label" type="Label" parent="ColorRect/Item_items_price/Button"]
-margin_left = -8.35962
-margin_top = -3.25937
-margin_right = 11.6404
-margin_bottom = 10.7406
-rect_scale = Vector2( 2, 2 )
+[node name="Button" type="Button" parent="Control/MarginContainer/VBoxContainer/CenterContainer/VBoxContainer/Item_items_price"]
+margin_left = 332.0
+margin_right = 352.0
+margin_bottom = 20.0
 text = "+"
-align = 1
-valign = 1
-
-[node name="Button" type="Button" parent="ColorRect"]
-margin_left = 231.49
-margin_top = 265.189
-margin_right = 348.49
-margin_bottom = 304.189
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="Button" type="Button" parent="Control/MarginContainer/VBoxContainer"]
```

Comment:
Can you use GameButton instead of Godot's default Button, for "EXIT"? (res://Scenes/UserInterface/Buttons/GameButton.tscn)